### DISCUSSION TITLE
Using results from alosStack.py for PSInSAR and SBAS analysis in StaMPS

### DISCUSSION THREAD

**Discussion by Daffodiles** [isce-framework/isce2#672] [Q&A]
Hello, I want to ask if I can use the results from alosStack processing for PSInSAR and SBAS analysis in StaMPS, and are there tutorials for the prepration of the results for StaMPS (if it can be done)?  Thank you very much!

**Comment by EJFielding** [comment]
I don't know of any tutorial for preparing the output from the `alosStack` workflow to analyze JAXA ALOS-2 data in StaMPS. There are very few tracks of ALOS-2 data with a large number of acquisitions in stripmap mode that would be best for PSInSAR analysis. The complicated structure of the ScanSAR mode processed with full-aperture focusing means that the ALOS-2 ScanSAR data is very unlikely to work well in StaMPS, even if you prepare the data correctly.
